srunner/tools/openscenario_parser.py

```diff
@@ -677,10 +677,7 @@ def convert_condition_to_atomic(condition, actor_list):
                     condition_rule = headtime_condition.attrib.get('rule')
                     condition_operator = OpenScenarioParser.operators[condition_rule]
 
-                    condition_freespace = strtobool(headtime_condition.attrib.get('freespace', False))
-                    if condition_freespace:
-                        raise NotImplementedError(
-                            "TimeHeadwayCondition: freespace attribute is currently not implemented")
+                    condition_freespace = strtobool(headtime_condition.attrib.get('freespace'))
```

Comment:
I'd keep the *False* there, as by default, if the attribute is not found, it returns None.